internal/commands/accesslist/update.go

```diff
@@ -0,0 +1,160 @@
+package accesslist
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/10gen/realm-cli/internal/utils/flags"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+type updateInputs struct {
+	cli.ProjectInputs
+	Address    string
+	NewAddress string
+	Comment    string
+}
+
+// CommandMetaUpdate is the command meta for the `accesslist update` command
+var CommandMetaUpdate = cli.CommandMeta{
+	Use:         "update",
+	Display:     "accesslist update",
+	Description: "Modify an IP address or CIDR block in the Access List of your Realm app",
+	HelpText: `Changes an existing entry from the Access List of your Realm app. You will be
+prompted to select an IP address or CIDR block to update if neither is
+specified.`,
+}
+
+// CommandUpdate is the `accesslist update` command
+type CommandUpdate struct {
+	inputs updateInputs
+}
+
+// Flags is the command flags
+func (cmd *CommandUpdate) Flags() []flags.Flag {
+	return []flags.Flag{
+		cli.AppFlagWithContext(&cmd.inputs.App, "to modify an entry in its Access List"),
+		cli.ProjectFlag(&cmd.inputs.Project),
+		cli.ProductFlag(&cmd.inputs.Products),
+		flags.StringFlag{
+			Value: &cmd.inputs.Address,
+			Meta: flags.Meta{
+				Name: "ip",
+				Usage: flags.Usage{
+					Description: "Specify the existing IP address or CIDR block that you would like to modify",
+				},
+			},
+		},
+		flags.StringFlag{
+			Value: &cmd.inputs.NewAddress,
+			Meta: flags.Meta{
+				Name: "new-ip",
```

Comment:
just a thought here (cc: @christineschen @sumedhamehta )

what do we think about adopting a naming strategy (it could be for flag names, future command names, etc) where instead of something like `"new-ip"`, we use `"ip-new"`?

i mention this b/c if/when auto-complete is ever supported, this "inverted" name would prove more useful

Reply:
should use `flagIPNew` as the `Name` here